skyportal/models/followup_request.py

```diff
@@ -203,3 +279,45 @@ def set_time(self, altitude=30 * u.degree):
     AccessibleIfUserMatches('followuprequest.requester')
     & FollowupRequestTargetGroup.read
 )
+
+
+@event.listens_for(Classification, 'after_insert')
```

Comment:
What do classifications have to do with this?

Reply:
In this first version, I am setting triggering criteria based on classification.